framework/encode/capture_manager.h

```diff
@@ -49,6 +49,10 @@
 GFXRECON_BEGIN_NAMESPACE(gfxrecon)
 GFXRECON_BEGIN_NAMESPACE(encode)
 
+class ApiCaptureManagerBase;
+class VulkanCaptureManager;
+class D3D12CaptureManager;
+
```

Comment:
It may be a good to make a stronger distinction between `ApiCaptureManagerBase` and `CaptureManager` and break with existing `CaptureManager`. For example, move `ApiCaptureManagerBase` to its own file and change the name of `CaptureManager` to `CommonCaptureManager` or `SharedCaptureManager`. The new capture manager classes (except for the shared Api class) could also be marked as final.

Reply:
Some more ideas for the name of the common capture manager class: `GlobalCaptureManager` or `CoreCaptureManager`